test/test_translator.py

```diff
@@ -232,26 +229,10 @@ def test_get_event_status():
 def test_wzdx_creator() :
     wzdx_schema = json.loads(open('translator/sample files/validation_schema/wzdx_v3.0_feed.json').read())
     icone_data =open('translator/sample files/Icone Data/incidents_extended.xml').read()
-
-
     icone_obj = xmltodict.parse(icone_data)
-
-    # info={}
-    # info['feed_info_id'] = "feed_info_id"
-    #
-    # #### This information is required, might want to hardcode
-    # info['metadata'] = {}
-    # info['metadata']['wz_location_method'] = "channel-device-method"
-    # info['metadata']['lrs_type'] = "lrs_type"
-    # info['metadata']['location_verify_method'] = "location_verify_method"
-    # info['metadata']['datafeed_frequency_update'] = 86400
-    # info['metadata']['timestamp_metadata_update'] = "timestamp_metadata_update"
-    # info['metadata']['contact_name'] = "contact_name"
-    # info['metadata']['contact_email'] = "contact_email"
-    # info['metadata']['issuing_organization'] = "issuing_organization"
-    test_wzdx=icone_translator.wzdx_creator(icone_obj,icone_translator.initialize_info())
-
-    validate(instance=test_wzdx,schema=wzdx_schema)
+    test_wzdx = icone_translator.wzdx_creator(icone_obj, icone_translator.initialize_info())
+    validate(instance=test_wzdx, schema=wzdx_schema)
+    assert True
```

Comment:
We shouldn't be asserting True, that doesn't prove our test accomplished anything. Rather, we should be validating that the returned object has expected properties. You opened a known file and parsed the xml. That means we know what the properties should look like on the returned object. Verify those properties instead, or we could have a function in here that returns a pre-built wzdx object that we could compare the returned object against.

Reply:
The validate function (from the jsonschema library) throws an exception if the validation fails. So if the validation succeeds, it will assert True and pass, if the validation fails then 
There are some random values created in the wzdx_creator, such as a UUID and two uses of datetime.now(). I do not know of a way to compare an object containing random values to a pre-build object that doesn't feel hacky, that is why we chose to use the validate function instead of equating the output.

Reply:
Validating the schema here is good, in that it is showing us the return value is a valid WZDx 3.0 schema. We don't need to `assert True` with that, but if we want to assert anything maybe we can assert no exception thrown? 

Either in this method or another unit test, this function returns an object that we expect to have certain properties and we should test for that. The test currently is just saying "any wzdx 3.0 message will do, regardless of the data I've given you". For instance, we are passing in the `icone_translator.initialize_info` as the "info" parameter. We could create a unit test version of this object (static so we know values), and verify that the output object contains expected values. IE our `test_wzdx` object has a feed_info_id set from that info, as well as some metadata, etc.

Also, we are currently reading in the `incidents_exended.xml` file and casting it to a dictionary with the `xmltodict.parse` method. Rather, we could have a known dictionary with data to test return values against.

Reply:
I used a regular expression to compare the generated wzdx message to the hard coded one.The reason i used the regular expression was to match the randomly generated  fields such as id and added some unity test edge cases for the function and pushed the updated code to dev branch.